### DISCUSSION TITLE
또 반성문...

### DISCUSSION THREAD

**Discussion by shkisme** [BDD-CLUB/challenge-every-week-posting#161] [6. 반성문]
저번주에... 글 썼는데 업로드하는 걸 까먹어버렸습니당... 😱
주말에 올려야지~ 했는데 역시 일을 미루면 안될 것 같습니다...ㅜ 반성반성...

[[BOJ] 1로 만들기](https://velog.io/@shkisme/1%EB%A1%9C-%EB%A7%8C%EB%93%A4%EA%B8%B0)
[[이코테] 개미 전사](https://velog.io/@shkisme/%EC%9D%B4%EC%BD%94%ED%85%8C-%EA%B0%9C%EB%AF%B8-%EC%A0%84%EC%82%AC)

이번 주 부터 친구들과 `지옥의 알고리즘 스터디`를 하게 되었는데요~ 강해져서 돌아오고 싶네요 ㅎㅎㅎㅎ

**Comment by pnudpwls1103** [comment]
지옥의 알고리즘 머찌다!! 우리 모두 강해져봐요 🦾
